src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class SortCommandParser implements Parser<SortCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
+     * and returns a FindCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public SortCommand parse(String args) throws ParseException {
+        String sortParam = ParserUtil.parseSort(args);
+        if (sortParam.contains("/n") || sortParam.contains("/i") || sortParam.contains("/m")) {
```

Comment:
Should switch case be used here instead? I noticed that the sort command will inform the user that a sort is performed but not perform any sorting (which might be unintended behaviour?) if either:
* more than one parameter is passed as argument to the command, or
* a string containing a paramter flag (eg /n, /i, or /m) is passed but contains other noise (eg. sort plan/index is accepted as a valid command as it contains /i, but no sorting is performed)